.buildkite/pipelines/security_solution/security_solution_cypress.yml

```diff
@@ -1,108 +1,96 @@
 steps:
-  - command: .buildkite/scripts/pipelines/security_solution_quality_gate/security_solution_cypress/mki_security_solution_cypress.sh cypress:run:qa:serverless
-    label: 'Serverless MKI QA Security Cypress Tests'
-    agents:
-      queue: n2-4-spot
-    # TODO : Revise the timeout when the pipeline will be officially integrated with the quality gate.
-    timeout_in_minutes: 300
-    parallelism: 6
-    retry:
-      automatic:
-        - exit_status: '*'
-          limit: 1
-
   - command: .buildkite/scripts/pipelines/security_solution_quality_gate/security_solution_cypress/mki_security_solution_cypress.sh cypress:run:qa:serverless:explore
-    label: 'Serverless MKI QA Explore - Security Solution Cypress Tests'
+    label: "Serverless MKI QA Explore - Security Solution Cypress Tests"
```

Comment:
Why are we chaning the `'` for a `"`? Taking a look at the different buildkite files everywhere it is used `'` why not be aligned?

Reply:
automatic formatting..... Let me handle this.

Reply:
Done